front/app/containers/Authentication/SuccessActions/actions/submitPoll.tsx

```diff
@@ -8,13 +8,11 @@ export interface SubmitPollParams {
   phaseId: string;
   answers: string[];
   projectId: string;
-  setIsSubmitting: (isSubmitting: boolean) => void;
 }
 
 export const submitPoll =
-  ({ phaseId, answers, projectId, setIsSubmitting }: SubmitPollParams) =>
+  ({ phaseId, answers, projectId }: SubmitPollParams) =>
   async () => {
-    setIsSubmitting(true);
```

Comment:
Removed this submit state, because it would only work in the case where the user entered the auth flow and did not leave the platform. But in that case the modal would be on top of the screen anyway, so not too much value in having a spinner somewhere in the background